fs/expose/dokan/__init__.py

```diff
@@ -89,22 +100,46 @@
 import logging
 logger = logging.getLogger("fs.expose.dokan")
 
+if six.PY3:
```

Comment:
Anyways, I think from

    from six.moves import range

at the top of the file is a lot cleaner, don't you think?